src/systemd/meson.build

```diff
@@ -66,3 +66,15 @@ python.install_sources(
         'test/test_id128.py',
         subdir: 'systemd/test',
 )
+
+include_dir = libsystemd_dep.get_variable(pkgconfig: 'includedir')
+
+update_constants = custom_target(
```

Comment:
Having bashed my head against meson for a bit now, shouldn't this be a `run_target` instead?